python/ray/rllib/a3c/runner.py

```diff
@@ -127,10 +143,13 @@ def env_runner(env, policy, num_local_steps, summary_writer, render):
       if render:
         env.render()
 
-      # Collect the experience.
-      rollout.add(last_state, action, reward, value_, terminal, last_features)
       length += 1
       rewards += reward
+      if length >= timestep_limit:
+        terminal = True
```

Comment:
Previously these weren't marked as terminal, which I think could cause reward values to be incorrectly totaled later on.